kubernetes/client/rest.py

```diff
@@ -193,7 +193,7 @@ def request(self, method, url, query_params=None, headers=None,
                 # Pass a `string` parameter directly in the body to support
                 # other content types than Json when `body` argument is
                 # provided in serialized form
-                elif isinstance(body, str) or isinstance(body, bytes):
+                elif isinstance(body, (str, bytes)):
```

Comment:
it is generated, please try fixing at the code generator.

================================
    No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)  # noqa: E501
    The version of the OpenAPI document: release-1.17
    Generated by: https://openapi-generator.tech

Reply:
Ah! okay. @yliaog